src/SSHLibrary/library.py

```diff
@@ -901,13 +901,22 @@ def login(self, username, password, allow_agent=False, look_for_keys=False, dela
         | `Open Connection` | linux.server.com | prompt=$         |
         | ${output}=        | `Login`          | johndoe          | secretpasswd |
         | `Should Contain`  | ${output}        | johndoe@linux:~$ |
+
+        Example that logs in through a proxy server (10.128.3.101)
+        | `Open Connection` | linux.server.com |
+        | ${output}=        | `Login`          | johndoe       | secretpasswd |
+/usr/bin/ssh -l robot -i ~/.ssh/robot_id_rsa -o StrictHostKeyChecking=no -o
```

Comment:
There are some issues with this example:
- it should be spanned only on a single line because when generating the keyword documentation the text will appear on multiple lines, breaking the table. You can use the ``\`` to indicate the statement will continue on the next line (if longer than 80 chars)
- the third argument of ``Login`` keyword is ``allow_agent``, you should use the it as a named argument, like ``proxy_command='command'``
- the example is very difficult to read and understand and it should be more generic. Is there a way to simplify this? And document exactly who ``h`` and ``p`` are? Like local host or remote host, or proxy (middle host). And do we really need that options like ``StrictHostKeyChecking`` and ``UserKnownHostsFile``? My feeling is that this example is tailored to your environment too much and maybe shortening as much as possible would make it more simple, like: ssh *user* -i *keyfile* -W *remote_host*:*remote_port* proxy.server.com. Would that make sense?